skimage/filters/thresholding.py

```diff
@@ -366,3 +367,97 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+def threshold_minimum(image, nbins=256, bias='min') :
+    """ Return threshold value based on minimum method
```

Comment:
Could you also add a 2-3 sentence description of what the algorithm does? People should not have to read the paper for a simple understanding.